JenkinsfileQARelease

```diff
@@ -6,17 +6,6 @@ node('linux && docker') {
       sh 'npm run setup'
     }
 
-    /**
-     * The build versions must be bumped before building ui-kit in order
-     * to expose the correct versions in headless and atomic.
-     */
-    stage('Graduate') {
-      withCredentials([
-      usernameColonPassword(credentialsId: 'github-commit-token', variable: 'GH_CREDENTIALS')]) {
-        sh 'npm run bump:version:graduate'
-      }
-    }
```

Comment:
This is what used to graduate the version to a non-alpha version. It's no longer necessary since all versions are non-alpha now.